app/tasks/maintenance_tasks/task.rb

```diff
@@ -110,5 +133,12 @@ def process(_item)
     # @return [Integer, nil]
     def count
     end
+
+    # Returns the list of the throttle specs for a Task instance.
+    #
+    # @return [Array<Hash{Symbol => Proc, ActiveSupport::Duration}>]
+    def throttle_specs
```

Comment:
Hum maybe actually we do want the instance-level to be public? That gives Task authors complete control over the throttling, and the class-level macro just makes it more convenient for the common use-cases?

Reply:
I think I'd prefer to leave this as private API for now until we're convinced that the class-level macro is insufficient for users.

Reply:
(I didn't intend this to be public API, just forgot to add the private tag 😅 )